ruskeyverbs/migrations/0005_auto_20180224_2126.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.0.2 on 2018-02-25 03:26
+
+from django.db import migrations
+from django.utils import timezone
+import random
+import datetime
+
+
+def populate_user(apps, schema_editor):
+    example_model = apps.get_model("ruskeyverbs", "Example")
+    user_model = apps.get_model("auth", "User")
+    performance_model = apps.get_model("ruskeyverbs", "PerformancePerExample")
+    my_user = user_model.objects.get(pk=1)
+    dictkey = 0
+    object_dict = {}
+    for i in range(1, example_model.objects.all().count()+1):
+        my_example = example_model.objects.get(id=i)
+        random_int = random.randint(1, 10)
+        random_int_days = random.randint(-5, -1)
+        study_date = timezone.now() + datetime.timedelta(days=random_int_days)
+        my_due_date = study_date + datetime.timedelta(days=random_int)
+        object_dict[dictkey] = performance_model(example=my_example,
```

Comment:
What is the point of object_dict and dictkey? You put the performance_model in there and save it right away. So why bother with the dict at all?

If you do need the dict, why do you need to compute dictkey like this? Wouldn't i always equal dictkey? Since i is your iterator it is safer to index off of that if you need to.

Reply:
Yeah, looking at this code I am really not sure what I was thinking in terms of using a dictionary. Maybe for some reason I was thinking I'd need the performance example objects for something else later on? In retrospect I am as perplexed as you are.